poland.py

```diff
@@ -68,6 +69,52 @@ def get_core(content):
     text_arr.append(normalize(item.get_text(strip=True, separator=' ')))
   return text_arr
 
+def configure_headers():
+  xml_settings_file = 'settings.xml'
+
+  #
+  # Check whether file exists on the system or not and whether it's a file or a directory.
+  #
+  if not path.exists(xml_settings_file) or not path.isfile(xml_settings_file):
+    return "The file doesn't exist on the system."
+
+  #
+  # Retrieve data from XML file and place it inside of a variable
+  #
+  with open(xml_settings_file, 'r') as file:
```

Comment:
Should probably load in chunks as a large file will probably fuck this part up

Reply:
Ok, i'll check that out.